Frontend/src/app/dispatching/dispatching-show/dispatching-show.component.css

```diff
@@ -24,3 +26,12 @@ mat-list-item {
     font-size: 0.9em;
     padding: 0 0 0 10px;
 }
+
+::ng-deep .mat-tab-label, ::ng-deep.mat-tab-label-active{
```

Comment:
Dont use `::ng-deep` without `:host`

For more info, read: https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep

Reply:
Done